File: Unstop/TCS/TomAndJerry.py
def find_cookie(stream, k):
    stream.sort(reverse=True)
    return stream[k - 1]

def main():
    n = int(input())
    k = int(input())
    stream = list(map(int, input().split()))

    cookie_position = find_cookie(stream, k)
    print(cookie_position)

if __name__ == "__main__":
    main()

File: Unstop/TCS/TomAndJerry.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int findCookie(vector<int>& stream, int k) {
    nth_element(stream.begin(), stream.begin() + k - 1, stream.end(), greater<int>());
    return stream[k - 1];
}

int main() {
    int n, k;
    cin >> n >> k;

    vector<int> stream(n);
    for (int i = 0; i < n; ++i) {
        cin >> stream[i];
    }

    int cookiePosition = findCookie(stream, k);
    cout << cookiePosition << endl;

    return 0;
}
